ecosystem/sep-datastore.md

```diff
@@ -0,0 +1,174 @@
+## Preamble
+
+```
+SEP: To Be Assigned
+Title: Ledger Metadata Storage
+Author: Tamir Sen <@tamirms>
+Status: Draft
+Created: 2025-03-11
+Version: 0.1.0
+```
+
+## Simple Summary
+
+A standard for how [`LedgerCloseMeta`](https://github.com/stellar/stellar-xdr/blob/v22.0/Stellar-ledger.x#L539-L545)
+objects should be stored so that ledgers can be easily and efficiently ingested by downstream systems.
+
+## Dependencies
+
+None.
+
+## Motivation
+
+[galexie](https://github.com/stellar/go/tree/master/services/galexie) is a service which publishes
+[`LedgerCloseMeta`](https://github.com/stellar/stellar-xdr/blob/v22.0/Stellar-ledger.x#L539-L545) XDR objects to a GCS
+(Google Cloud Storage) bucket. However, the data format and layout of the XDR objects are not formally documented. This
+SEP aims to provide a comprehensive specification for storing LedgerCloseMeta objects, enabling third-party developers
+to build compatible data stores and clients for retrieving ledger metadata.
+
+## Specification
+
+The data store is a key-value store where:
+
+- **Keys** are strings following a specific hierarchical format.
+- **Values** are binary blobs representing compressed `LedgerCloseMetaBatch` XDR values.
+
+The key-value store must support:
+
+- Efficient point lookups for arbitrary keys.
+- Listing keys in lexicographic order, optionally filtered by a prefix.
+
+Examples of compatible key-value stores include Google Cloud Storage (GCS) and Amazon S3.
```

Comment:
maybe mention O/S File System here as well, to highlight it's not cloud dependent?